src/hooks/useHasFocus/useHasFocus.stories.mdx

```diff
@@ -0,0 +1,28 @@
+import { Meta } from '@storybook/blocks';
+
+<Meta title="hooks/useHasFocus" />
+
+# useHasFocus
```

Comment:
The hook itself isn't related to `document.activeElement`. I don't think we should explain the difference between the hook and the `document.activeElement` when they are not directly related, it could be confusing the reader.

Reply:
Aren't both for detecting focus? Or at least, it might be useful for people to know how those pseudo classes work. Like a short explanation and a link to MDN.

Reply:
`document.activeElement` returns the element that currently has focus, this is afaik the same as `:focus`. I don't think we should explain the pseudo selectors, there are other resources for this that do this better than we can.